test/integration/dataContract/DataContractLevelDBRepository.spec.js

```diff
@@ -24,12 +25,12 @@ describe('DataContractLevelDBRepository', () => {
     dppMock = createDPPMock(this.sinon);
     dppMock
       .dataContract
-      .createFromSerialized
+      .createFromBuffer
       .resolves(dataContract);
 
     repository = new DataContractLevelDBRepository(db, dppMock);
 
-    key = `${DataContractLevelDBRepository.KEY_PREFIX}:${dataContract.getId()}`;
+    key = `${DataContractLevelDBRepository.KEY_PREFIX}:${bs58.encode(dataContract.getId())}`;
```

Comment:
You can put just `dataContract.getId()` and `toString()` will be called automatically.